spec/java/util/stream/Stream.Spliterator.lsl

```diff
@@ -0,0 +1,182 @@
+libsl "1.1.0";
+
+library std
+    version "11"
+    language "Java"
+    url "https://github.com/openjdk/jdk11/blob/master/src/java.base/share/classes/java/util/Spliterators.java";
+
+// imports
+
+import java/util/stream/Stream;
+import java/util/function/Consumer;
+import java/util/Spliterator;
+import java/util/Comparator;
+
+
+// automata
+
+automaton StreamSpliteratorAutomaton
```

Comment:
Maybe it is worth adding a comment about it is an ArraySpliterator

Reply:
Was added;